src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py

```diff
@@ -2149,101 +1858,6 @@ def test_import_zaaktype_invalid_eigenschap(self, *mocks):
         self.assertEqual(ZaakType.objects.count(), 0)
         self.assertEqual(Eigenschap.objects.count(), 0)
 
-    def test_import_zaaktype_informatieobjectype_overlapping(self, *mocks):
```

Comment:
I'm not sure what this test was testing to begin with, it's no longer possible now to export/import the same data without removing the existing data, so I removed it